hack/verify-structured-logging.sh

```diff
@@ -48,15 +48,33 @@ while IFS='' read -r line; do
   migrated_packages+=("$KUBE_ROOT/$line")
 done < <(cat "${migrated_packages_file}")
 
-
+echo "Running structured logging static check on migrated packages"
 ret=0
 GOOS=linux    logcheck "${migrated_packages[@]}" || ret=$?
 GOOS=windows  logcheck "${migrated_packages[@]}" || ret=$?
 
 if [ $ret -eq 0 ]; then
-  echo "Structured logging static check is passed :)."
+  echo "Structured logging static check passed on migrated packages :)"
+else
+  echo "Please fix above failures. You can locally test via:"
+  echo "hack/verify-structured-logging.sh"
+fi
+
+# Ignore migrated packages as they are already tested
+# Trim the trailing /... from given packages
+ignore_packages=`cat ${KUBE_ROOT}/hack/.structured_logging | grep -oE '^[a-zA-Z0-9/]+[^/\.]'`
+# Packages to test = all packages - ignored packages
+packages=`go list  ${KUBE_ROOT}/... | grep -v '$ignore_packages'`
```

Comment:
I don't think this works, or at least when I tried it. Can you check this
```suggestion
packages=`grep -vf <(cat ./hack/.structured_logging | grep -oE '^[a-zA-Z0-9/]+[^/\.]') <(go list ./...) `
```

Reply:
Yes, it didn't work and I made the suggested changes.